src/tidy.zig

```diff
@@ -361,7 +361,16 @@ fn find_long_line(file_text: []const u8) !?usize {
             if (parse_multiline_string(line)) |string_value| {
                 const string_value_length = try std.unicode.utf8CountCodepoints(string_value);
                 if (string_value_length <= 100) continue;
+
+                if (std.mem.startsWith(u8, string_value, " account A") or
+                    std.mem.startsWith(u8, string_value, " transfer T") or
+                    std.mem.startsWith(u8, string_value, " transfer   "))
+                {
+                    // Table tests from state_machine.zig. They are intentionally wide.
```

Comment:
The general philosophy is that we don't have a fine grained opt-out like `fmt:off`, and instead add extremely well motivated exceptions to the tidy itself. 

I think our state machine tests pass the threshold of well-motivatedness.